Task_1/Parts/Class.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_1
+{
+    class Class
+    {
+        Pupil[] pupils = new Pupil[4];
```

Comment:
Private fields should start with '_'. It's kinda standard
```suggestion
        Pupil[] _pupils = new Pupil[4];
```